src/program/OutfitStudio.cpp

```diff
@@ -8932,15 +8936,12 @@ void wxGLPanel::EndPivotPosition() {
 	std::vector<mesh*> refMeshes = activeStroke->GetRefMeshes();
 	if (refMeshes.size() > 0) {
 		mesh* pivotHitMesh = refMeshes[0];
```

Comment:
"pivotHitMesh" (line 8938) is now unused. Can the line be removed?

Reply:
Yes.  Shall I do it?